libs/emblem/src/components/market-emblem.tsx

```diff
@@ -42,6 +42,7 @@ export function EmblemByMarket(props: EmblemByMarketProps) {
   // Widths are calculated here as they are required for using absolute positioning to
   // render the logos as overlapping. Moving to blocks with negative margins should work
   // and could remove this calculation, but was not working reliably cross platform.
+  // const wrapperClassString = classNames('relative flex items-center', {
   const wrapperClassString = classNames('relative inline-block h-6', {
     'w-8': logoCount === 2,
```

Comment:
```suggestion
    'w-9': logoCount === 2,
```

`w-8` is too narrow for the two logos side by side, which can cause wrapping in some places